### PR TITLE
update network module tests for pipeline

### PR DESCRIPTION
**Pull request by TheOnlyWei**
Here is the pipeline result. I skipped the tests for get-azsnetworkquota because the command is currently bugged.

https://dev.azure.com/azure-sdk/internal/_build/results?buildId=420492&view=results

### CODE REVIEW

File: src/Azs.Network.Admin/test/Common.ps1 [hunk 1]
```diff
@@ -15,11 +15,14 @@
 # Tests to skip
 $global:SkippedTests = $(
     'TestGetAllVirtualNetworksOData',
-    'TestGetAllPublicIpAddressesOData'
+    'TestGetAllPublicIpAddressesOData',
```
Comment (viananth) [inline src/Azs.Network.Admin/test/Common.ps1:18]: Why are these tests skipped? Please create issues and assign to network team.

File: src/Azs.Network.Admin/test/Get-AzsLoadBalancer.Tests.ps1 [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+    $envFilePath = Join-Path $PSScriptRoot $envFile
+} else {
+    $envFilePath = Join-Path $PSScriptRoot '..\$envFile'
+}
+$env = @{}
+if (Test-Path -Path $envFilePath) {
+    $env = Get-Content (Join-Path $PSScriptRoot $envFile) | ConvertFrom-Json
+    $PSDefaultParameterValues = @{"*:SubscriptionId" = $env.SubscriptionId; "*:Tenant" = $env.Tenant; "*:Location" = $env.Location; "*:ResourceGroupName" = $env.ResourceGroup }
+}
+
```
Comment (viananth) [inline src/Azs.Network.Admin/test/Get-AzsLoadBalancer.Tests.ps1:16]: Remove this section from all tests and add default parameters for location and resource group name in utils.ps1 setEnv function.

File: src/Azs.Network.Admin/test/Common.ps1 (hunk 1 above)
Comment (TheOnlyWei) [reply src/Azs.Network.Admin/test/Common.ps1:18]: We can make a new task to go over all skipped tests. For now I think we should just commit working pipeline tests.

Comment (bganapa) [reply src/Azs.Network.Admin/test/Common.ps1:18]: Can you pls explain the reason for skipping? we can involve the network team and get them fixed

Comment (TheOnlyWei) [reply src/Azs.Network.Admin/test/Common.ps1]: Emailed Joshua Ya about this, CCed reviewers.

File: src/Azs.Network.Admin/test/Get-AzsLoadBalancer.Tests.ps1 (hunk 2 above)
Comment (TheOnlyWei) [reply src/Azs.Network.Admin/test/Get-AzsLoadBalancer.Tests.ps1:2]: Removed and created a loadEnvJson.ps1 to be imported.

**Pull request merged**